src/main/features/eligibility/store.ts

```diff
@@ -24,10 +24,10 @@ export class CookieEligibilityStore {
       }
       return undefined
     }
-    res.cookie(cookieName, _.cloneDeepWith(eligibility, excludeDisplayValue), { httpOnly: true, secure: true, sameSite: 'lax', maxAge: cookieTimeToLiveInMinutes })
+    res.cookie(cookieName, _.cloneDeepWith(eligibility, excludeDisplayValue), { httpOnly: true, secure: true, sameSite: false, maxAge: cookieTimeToLiveInMinutes })
```

Comment:
just remove sameSite from  here or set to 'none' i think

Reply:
hmm docs say false will work
https://www.npmjs.com/package/cookies

Reply:
false broke all the unit tests